src/main/java/codeu/model/data/BotRequest.java

```diff
@@ -0,0 +1,35 @@
+package codeu.model.data;
+
+import java.util.UUID;
+
+public class BotRequest {
+  private final String command;
+  private final BotArguments arguments;
```

Comment:
Since `BotArguments` is just a wrapper around an `ArrayList`, do you really need this indirection?

Could this just be `private final ArrayList<String> arguments`? Are you trying to allow for us to have more stuff in `BotArguments` in the future? What's the reasoning for this pretty much?

Reply:
I agree with this idea, BotArguments seems unnecessary. However, the arguments may not be `String` types, so I'm trying to use an ArrayList of Objects.